packages/wonder-stuff-ci/package.json

```diff
@@ -6,7 +6,7 @@
         "node": ">=16"
     },
     "name": "@khanacademy/wonder-stuff-ci",
-    "version": "1.1.0",
+    "version": "1.0.0",
```

Comment:
Err, that doesn't seem right! I think you have to revert this change - the version changes will come as part of the Changesets release process.